updater/omahaHandler_test.go

```diff
@@ -0,0 +1,42 @@
+package updater
```

Comment:
Ditto blackbox testing.
```suggestion
package updater_test
```